address/Cargo.toml

```diff
@@ -0,0 +1,87 @@
+[package]
+name = "solana-address"
+description = "Solana account addresses"
+documentation = "https://docs.rs/solana-address"
+version = "0.1.0"
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[features]
+atomic = ["dep:solana-atomic-u64"]
+borsh = ["dep:borsh", "std"]
+bytemuck = ["dep:bytemuck", "dep:bytemuck_derive"]
+curve25519 = ["dep:curve25519-dalek", "error", "sha2"]
+decode = ["dep:five8", "dep:five8_const", "error"]
+dev-context-only-utils = ["dep:arbitrary", "rand"]
+error = ["dep:num-traits", "dep:solana-program-error"]
+frozen-abi = [
+    "dep:solana-frozen-abi",
+    "dep:solana-frozen-abi-macro",
+    "dep:solana-program-error",
+    "std",
+]
+rand = ["dep:rand", "atomic", "std"]
+sanitize = ["dep:solana-sanitize"]
+serde = ["dep:serde", "dep:serde_derive"]
+sha2 = ["dep:solana-sha256-hasher", "error"]
+std = ["decode"]
+syscalls = ["dep:solana-define-syscall", "error"]
```

Comment:
The "error" feature is needed for syscalls since the helpers return an `AddressError` (used to be `PubkeyError`) instead of a `ProgramError`. I think the differentiation is there since these helpers can be used offchain. We could make the on-chain "version" of those helpers return a `ProgramError` so programs don't need the "error" feature in a future PR.